### PR TITLE
First Samudralaya Release [v1.0]

### PR DESCRIPTION
**Pull request by muhasyraf**

### CODE REVIEW

File: resources/js/Layouts/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import ApplicationLogo from "@/Components/ApplicationLogo";
+import { Link } from "@inertiajs/react";
+
+export default function Footer() {
+    return (
+        <footer className="bg-bohim-snow max-w-screen-2xl mx-auto pt-20">
```
Comment (sdmalfian) [inline resources/js/Layouts/Footer.jsx]: please reduce footer padding-top to 12 (pt-12)

File: resources/js/Pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import Articles from "@/Components/Home/Articles";
+import Report from "@/Components/Home/Report";
+import AuthenticatedLayout from "@/Layouts/AuthenticatedLayout";
+import { Head } from "@inertiajs/react";
+
+export default function Home(props) {
+    return (
+        <AuthenticatedLayout user={props.auth.user}>
+            <Head title="Home" />
+            <div className="mt-12 mb-10">
```
Comment (sdmalfian) [inline resources/js/Pages/Home.jsx]: please remove margin-bottom class

**Review by sdmalfian — CHANGES_REQUESTED** [review]

File: resources/js/Layouts/Footer.jsx (hunk 1 above)
Comment (muhasyraf) [reply resources/js/Layouts/Footer.jsx]: alright sir, i'll do it immediately

File: resources/js/Pages/Home.jsx (hunk 2 above)
Comment (muhasyraf) [reply resources/js/Pages/Home.jsx]: with pleasure sir, i'm on it

**Review by sdmalfian — APPROVED** [review]

**Pull request merged**